packages/compass-indexes/src/stores/store.ts

```diff
@@ -114,6 +129,8 @@ const configureStore = (options: ConfigureStoreOptions) => {
   }
 
   void store.dispatch(fetchIndexes());
+  void store.dispatch(fetchSearchIndexes());
```

Comment:
This i think we should move to UI. And based on the active tab, dispatch the correct initial action. And this action maybe will also introduce `isLoading` on regular indexes slice and that way we can align both search/regular indexes (or maybe add status there as well and remove these flag?)